app.rb

```diff
@@ -64,27 +86,24 @@ def create_book
   # CREATE A RENTAL METHOD
   def create_rental
     puts 'Select a book from the following list by number'
-    @books.each_with_index do |book, index|
-      puts "#{index}) Title: '#{book.title}', Author: #{book.author}"
-    end
-    book_input = gets.chomp.to_i
+    list_books
+    book_number = gets.chomp.to_i
 
     puts 'Select a person from the following list by number'
-    @people.each_with_index do |person, index|
-      puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
-    end
-    person_input = gets.chomp.to_i
+    list_people
+    person_number = gets.chomp.to_i
 
     print 'Date: '
     date = gets.chomp
 
-    rental = Rental.new(date, @books[book_input], @people[person_input])
+    @rentals.push(Rental.new(date, @people[person_number - 1], @books[book_number - 1]))
     @rentals.push(rental)
-    puts 'Rental created successfully!'
+
+    puts 'Book rented successfully'
   end
 
   # LIST ALL RENTALS FOR A GIVEN PERSON ID
-  def list_all_rentals
+  def list_rentals
     print 'Enter Person\'s ID: '
     id = gets.chomp.to_i
     puts 'List of all Rentals books: '
```

Comment:
- [ ] Great job working on the Refactoring but I think while refactoring your code you had error here when listing the Rentals when I tried to list the rentals by the id of the person I think it can't see the Id well so kindly make sure that it is working properly 🙏🏻😊💪🏻🚀✨


![image](https://user-images.githubusercontent.com/58553711/200448513-12fff9b2-4de4-4d08-bf86-42e00c76bfea.png)